src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.ts

```diff
@@ -164,6 +164,21 @@ export class SavedScenariosComponent implements OnInit {
     return isValidTotalArea(this.plan.area_acres);
   }
 
+  openConfirmationDialog(): void {
+    this.dialog
+      .open(ScenarioCreateConfirmationComponent, {})
+      .afterClosed()
+      .subscribe((response: any) => {
+        if (response) {
+          this.goToTreatmentPlans();
+        }
+      });
+  }
+
+  goToTreatmentPlans(): void {
+    // TODO: route to next page
```

Comment:
not sure if you wanna handle this on the next pr, but here we need something similar to https://github.com/OurPlanscape/Planscape/blob/main/src/interface/src/app/plan/create-scenarios/treatments-tab/treatments-tab.component.ts#L42

Reply:
Yeah, let's talk about this one, especially if there are backend things that can automatically happen here.